src/cc2olx/olx.py

```diff
@@ -250,6 +252,18 @@ def process_external_tools_link(item, html):
             html = html.replace(item, external_tool_url)
             return html
 
+        def process_extra_static_files(item, html):
+            if self.relative_links_source is None:
+                return html
+
+            for static_file in self.cartridge.extra_static_files:
```

Comment:
How large can `extra_static_files` get? This looks like it could get pretty expensive to loop through every entity in this list for each item.

Reply:
I might be misunderstanding this at a basic level, but here is how I interpret this:

1. There is a shared list of `extra_static_files`.
2. There are many items.
3. This is lookup n^2, i.e. (number of items) * (number of extra static files).
4. Each static file maps to the same OLX static path every time.

If that is accurate, then could you initialize a dict that has the mapped values, where the keys are `OLX_STATIC_PATH_TEMPLATE.format(static_filename=static_file)` and the values are the `static_file`? That way, the lookup here is just `if item in {my lookup dict}`, i.e. the whole thing goes from n^2 to n.

The thing is, if `extra_static_files` is small, this doesn't matter at all. In the past this sort of thing has been a problem when the list grew to thousands.

Reply:
https://github.com/openedx/edx-platform/pull/11095#issuecomment-173274286 is an example of something like this that has come up before.

Reply:
It's refactored and the bug with making `web_resources` static links absolute is fixed.